app/src/main/java/com/yeonae/chamelezone/view/home/HomeTabFragment.kt

```diff
@@ -102,99 +98,61 @@ class HomeTabFragment :
-
-                        placeAdapter.setItemClickListener(object :
-                            HomePlaceRvAdapter.OnItemClickListener {
-                            override fun onItemClick(
-                                view: View,
-                                position: Int,
-                                place: PlaceResponse
-                            ) {
-                                placeNumber = place.placeNumber
-                                val intent =
-                                    Intent(requireContext(), PlaceDetailActivity::class.java)
-                                intent.putExtra(PLACE_NAME, place.name)
-                                intent.putExtra(PLACE_NUMBER, place.placeNumber)
-                                startActivity(intent)
-                            }
-                        })
-
-                        placeAdapter.setLikeButtonListener(object :
-                            HomePlaceRvAdapter.LikeButtonListener {
-                            override fun onLikeClick(
-                                placeResponse: PlaceResponse,
-                                isChecked: Boolean
-                            ) {
-                                placeNumber = placeResponse.placeNumber
-
-                                if (memberNumber == 0) {
-                                    val intent = Intent(context, LoginActivity::class.java)
-                                    startActivity(intent)
-                                } else {
-                                    if (isChecked) {
-                                        presenter.selectLike(memberNumber, placeNumber)
-                                    } else {
-                                        presenter.deleteLike(memberNumber, placeNumber)
-                                    }
-                                }
-                            }
-                        })
-
-                        recycler_view_place?.apply {
-                            layoutManager = GridLayoutManager(context, 2)
-                            if (::placeAdapter.isInitialized)
-                                adapter = placeAdapter
-                        }
+                    if (isChecked) {
+                        presenter.selectLike(memberNumber, placeNumber)
                     } else {
-                        myLocationHandler.postDelayed(this, 1000)
+                        presenter.deleteLike(memberNumber, placeNumber)
                     }
                 }
             }
-        }, 500)
+        })
 
-        presenter = HomePresenter(
-            Injection.placeRepository(), Injection.memberRepository(),
-            Injection.likeRepository(),
-            this
-        )
+        recycler_view_place?.apply {
+            layoutManager = GridLayoutManager(context, 2)
+            if (::placeAdapter.isInitialized)
+                adapter = placeAdapter
+        }
```

Comment:
```suggestion
        recycler_view_place?.run {
            layoutManager = GridLayoutManager(context, 2)
            if (::placeAdapter.isInitialized)
                adapter = placeAdapter
        }
```
apply 보다는 run이 적합할 것으로 보이고, null check를 하지 않아도 될 것같습니다.

Reply:
수정했습니다
b8d9586